src/info/zcoin.js

```diff
@@ -26,7 +26,7 @@ const bcoinInfo: BcoinCurrencyInfo = {
 
 const engineInfo: EngineCurrencyInfo = {
   network: 'zcoin',
```

Comment:
I believe the network name needs to change to "firo". Confirm with @thehobbit85, but I believe this affects the address URI format

Reply:
I think the simpler solution is to use uriPrefix. Changing the network name requires additional changes to work that I haven't investigating besides seeing that the wallets don't load with network = firo. Using uriPrefix comes with the side benefit of being able to scan both zcoin: and firo: uris

Reply:
I was not aware of URI prefix. If that is available then yes definitely use it